src/Makefile.am

```diff
@@ -30,6 +32,12 @@ EXTRA_DIST = mp.pl
 mp.S	: mp.pl
 	./mp.pl mp.S
 
+if ICA_FIPS
+all-local: 
```

Comment:
This is still very dangerous.  In a parallel build, there is no guarantee that libica.so is already build when this rule runs.  As a bare minimum, this rule should depend on $(LTLIBRARIES) to make sure, libtool has built the libraries.

Reply:
Hm, yes, didn't think of parallel builds yet. I added the $(LTLIBRARIES) dependency and verified that e.g. a make -j8 succeeds.